idl/utils.idl.hh

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright 2023-present ScyllaDB
+ */
+
+/*
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+#include "utils/tagged_integer.hh"
+
+namespace utils {
+
+template<typename Tag, typename ValueType>
+struct tagged_integer final {
```

Comment:
`tagged_uint64` that this comes to replace was not marked as `final` (unfortunately), so you broke backwards compatibility here. Likely causing this: https://github.com/scylladb/scylladb/issues/13752

Reply:
sigh :-/

https://github.com/scylladb/scylladb/pull/13774 should fix this.